### PR TITLE
feat : added installation guide

### PR DESCRIPTION
**Pull request by ArjunJunna**
added installation guide to the library.

### CODE REVIEW

File: get started/introduction.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+    <link rel="stylesheet" href="../../style.css" />
+    <title>Introduction</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <div class="left-nav">
+          <a href="#">Aqua UI</a>
+        </div>
```
Comment (vrinda-mahajan) [inline get started/introduction.html]: Provide link to the index.html page here.

**Review by vrinda-mahajan — APPROVED** [review]
Good to merge.

File: get started/introduction.html (hunk 1 above)
Comment (ArjunJunna) [reply get started/introduction.html]: sure thank you

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -62,7 +62,7 @@ <h3>Below is an example for Image Avatar</h3>
           </div>
           <ul class="list-subtopic">
             <li>
-              <a href="#"> Introduction </a>
+              <a href="/get started/introduction.html"> Introduction </a>
```
Comment (keshavkumar-in) [inline components/avatar/avatar.html]: You should not use spaces in the folder name

File: css/layout/navigation.css [hunk 3]
```diff
@@ -16,6 +16,7 @@ nav {
 .left-nav a {
   color: #333;
```
Comment (anik31) [inline css/layout/navigation.css]: suggestion - use color variables

File: get started/introduction.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        &lt;link rel=&quot;stylesheet&quot; href=&quot;https://its-aqua-ui.netlify.app/components/components.css&quot;&gt;
+            </code>
+          </pre>
+        </div>
+      </main>
+      <div class="sidebar">
+        <!-- Get Started List Start -->
+        <div>
+          <div class="list-topic">
+            <a href="#"> Get Started </a>
```
Comment (keshavkumar-in) [inline get started/introduction.html]: Use `javascript:;` inplace of `#`

**Review by keshavkumar-in — APPROVED** [review]
Just had a few suggestions, other than that look good to me

**Review by anik31 — COMMENTED** [review]
Nice UI & clean code 💯

File: css/layout/navigation.css (hunk 3 above)
Comment (ArjunJunna) [reply css/layout/navigation.css]: sure will make the changes. thank you

File: components/avatar/avatar.html (hunk 2 above)
Comment (ArjunJunna) [reply components/avatar/avatar.html]: sure will make the changes thank you

**Pull request merged**